src/Evolution/Systems/Cce/Tags.hpp

```diff
@@ -404,5 +405,9 @@ struct InterpolationManager : db::SimpleTag {
     return "InterpolationManager(" + db::tag_name<ObservationTag>() + ")";
   }
 };
+
+struct GHInterfaceManager : db::SimpleTag {
```

Comment:
GhInterfaceManager;  also get rid of the identically named InitializationTag above and just keep it in the DataBox at the end of initialization